examples/simple-game-server/helper.go

```diff
@@ -0,0 +1,331 @@
+package main
```

Comment:
Rename suggestions : `sdk.go` (don't love it) - but `helpers.go` is a bit ambiguous. `sdk.go` gives some idea that this is where the sdk gets called.

Either that or pull it all into `handers.go` and put the SDK command under each handler? (or if need be, make it inline?)

Reply:
> Rename suggestions : `sdk.go` (don't love it) - but `helpers.go` is a bit ambiguous. `sdk.go` gives some idea that this is where the sdk gets called.
> 
> Either that or pull it all into `handers.go` and put the SDK command under each handler? (or if need be, make it inline?)

I have pulled everything inside `handlers.go`. sdk.go isn't required.
Thank you so much for your guidance.